.agents/evals/run.sh

```
@@ -0,0 +1,178 @@
+echo ""
+
+# ─── Phase 1: Build & Lint ───
+echo "${BLUE}━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━${NC}"
+echo "${BLUE}🔨 PHASE 1: Build & Lint${NC}"
+echo "${BLUE}━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━${NC}"
+echo ""
+
+# Frontend build
+if [[ -d "${REPO_ROOT}/frontend/plugins/${FRONTEND_PLUGIN}" ]]; then
+  echo "Building frontend: ${FRONTEND_PLUGIN}..."
+  if pnpm nx build "${FRONTEND_PLUGIN}" > /tmp/eval-build-${FRONTEND_PLUGIN}.log 2>&1; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)